pitchperfect/ipitchperfect
# Feat/video  Add Video storage and retrieval endpoints

To Test:
- Using Postman GET
  http://s1pitchperfect.azurewebsites.net/api/videos/url/53e4115151265a6cc67bf9a0
- Should return secure url to video on Azure
- Copy that URL into FireFox browser
- Watch the video

Details:
/videos POST
Create Video document in Mongo
Return video _id

/videos/url/:video_id  GET
Return secure access URL to video on Azure
